rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "stable"
```

Comment:
So, the downside of specifying stable is that CI will break every time rust releases a new version and adds additional warnings. The upside is that you instantly get notified of new warnings/issues, it can just be a little annoying when you're trying to merge a PR and suddenly CI decides to break for unrelated reasons).